sw/device/silicon_creator/lib/cert/cdi_1.hjson

```diff
@@ -46,6 +46,10 @@
             exact-size: 4,
             tweak-msb: true,
         }
+        // Debug (whether LC state exposes JTAG access or not).
```

Comment:
Technically a chip could be in the PROD Hardware LC state, but boot owner firmware signed with a TEST/DEV application key. This would mean JTAG access is not exposed.
```suggestion
        // Debug (whether owner FW is signed with PROD or DEV/TEST keys).
```